ads/google/doubleclick.js

```diff
@@ -164,6 +172,12 @@ function doubleClickWithGlade(global, data, gladeExperiment) {
   if (gladeExperiment === GladeExperiment.GLADE_EXPERIMENT) {
     jsonParameters.gladeExp = '1';
   }
+  const expIds = data['experimentId'];
+  if (expIds) {
+    jsonParameters.gladeEids = jsonParameters.gladeEids ?
+        jsonParameters.gladeEids + ',' + expIds : expIds;
```

Comment:
Yes, we've coordinated. For reference, you can see how he intends to use this parameter on his PR: https://github.com/ampproject/amphtml/pull/4688.